lib/next-amp-demo/next-env.d.ts

```diff
@@ -0,0 +1,2 @@
+/// <reference types="next" />
```

Comment:
Should this file have a copyright message? (Your call.)

Reply:
:-) I think we can skip this one  - it's auto generated as well